### ISSUE TITLE
Feature request to add 2 IO ports for S0 signals.

### ISSUE THREAD

**Issue by Roos-AID** [rvdbreemen/DSMRloggerAPI#1]
Hi Robert,

Great work with the Influxdb addition. 

A suggested improvement is to add functionality to measure counts on 2 IO so that s0 pulses from kWh meters can be recorded. 
Every pulse should increment a counter for that port.  The counter should be able to be set to a seed value to sync the counter with the total value on the kWh meter.  By measuring the time difference between 2 counts, it would be possible to indicate the current kW production or usage.

The data could then be used to track the gross energy production from solar panels and the consumption of  heatpump.

**Comment by rvdbreemen** [comment]
@Roos-AID yep, great idea. This Feature Request can be added to my list of todo's.

**Comment by MaartenSanders** [comment]
Hi all,
Running DSMRlogger-Next I am having some strange values in my RingHours.csv file. Even after a couple of days running. Does anyone have a suggestion what to do to resolve?
Thanks, Maarten

YYMMDDHH;      EDT1;      EDT2;      ERT1;      ERT2;       GDT;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
20120322;  2921.689;  2057.098;     1.123;     4.488;   903.310;         ;
20120323;  2923.484;  2057.098;     1.123;     4.488;   904.060;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
20120221;  2911.033;  2046.357;     1.123;     4.488;   894.730;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
20120223;  2914.495;  2046.357;     1.123;     4.488;   895.680;         ;
20113023;  2895.260;  2020.647;     1.123;     4.211;   874.920;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
00000000;     0.000;     0.000;     0.000;     0.000;     0.000;         ;
